drivers/SmartThings/matter-lock/src/init.lua

```diff
@@ -21,8 +21,11 @@ local PowerSource = clusters.PowerSource
 local capabilities = require "st.capabilities"
 local im = require "st.matter.interaction_model"
 local lock_utils = require "lock_utils"
+local device_lib = require "st.device"
 
 local INITIAL_COTA_INDEX = 1
+local AQARA_MANUFACTURER_ID = 0x115f
```

Comment:
@HunsupJung can we just add a fingerprint for this device instead? We could create a fingerprint that matches these devices to the `lock-without-codes` profile and potentially avoid any driver changes.

Reply:
Could you please add the corresponding fingerprint to the `fingerprints` file since it will be required for WWST anyway?

Reply:
I'm sorry for the late review. I will add fingerprint.